src/hooks/useFlags/featureFlagsProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { FlagsmithProvider } from "./flagsmith/flagsmithProvider";
+
+interface FeatureFlagsProviderProps {
+  children: React.ReactElement;
+}
+
+export const FeatureFlagsProvider = ({
+  children,
+}: FeatureFlagsProviderProps) => {
+  const isFeatureFlagsEnabled = process.env.FLAGS_SERVICE_ENABLED === "true";
+
+  return (
+    <FlagsmithProvider preventFetch={!isFeatureFlagsEnabled}>
```

Comment:
if flag service is not enabled you should render children directly, there is no need to render that provider if it's not in use